docs/source/security/cors.mdx

```diff
@@ -3,9 +3,7 @@ title: Configuring CORS
 description: Control access to your server's resources
 ---
 
-> ⚠️ New in Apollo Server 3.7: we **highly recommend** that all users pass `csrfPrevention: true` to `new ApolloServer()` to protect your server from CSRF and XS-Search attacks. **This is especially important if you integrate `graphql-upload` into your server.** If you enable this feature, any clients that send operations via `GET` requests or multipart upload requests must include a non-empty `Apollo-Require-Preflight` (or any other non-special-cased) header in that request. For more information, see [Preventing Cross-Site Request Forgery (CSRF)](#preventing-cross-site-request-forgery-csrf).
-
-> ⚠️ `graphql-upload` has a known CSRF vulnerability and is turned on by default in Apollo Server 2, specifically versions `< 2.25.4`. If you are using Apollo Server 2 and do _not_ use uploads, you can prevent this by disabling uploads by passing `uploads: false` to `new ApolloServer()` or by upgrading to version `^2.25.4`. If you _do_ use uploads, you **must** update to Apollo Server 3.7 or later and enable CSRF prevention in order to use this feature safely.
+> 📣 By default, Apollo Server 4 ships with a feature protecting from CSRF and XS-Search attacks. This feature requires that any client sending operations via `GET` or multipart upload requests _must_ include a non-empty `Apollo-Require-Preflight` (or another non-special-cased) header in that request. For more information, see [Preventing Cross-Site Request Forgery (CSRF)](#preventing-cross-site-request-forgery-csrf).
```

Comment:
protecting who from attacks? "protecting your users" perhaps?

I think the parenthetical is confusing, even though technically correct. Maybe "_must_ include a special header (such as `Apollo-Require-Preflight`) in their requests"

Reply:
1. For your first point, how does "protects a server's users" feel instead?
> By default, Apollo Server 4 ships with a feature that protects a server's users from CSRF and XS-Search attacks.

Reply:
sounds good